exercise_3/src/Course.cpp

```diff
@@ -0,0 +1,23 @@
+#include <stdexcept>
+#include <sstream>
+#include <string>
+#include "Course.h"
+
+Course::Course(std::istringstream &record) {
```

Comment:
Please consider to create init(), which do heavy work from constructor body. Or maybe do it outside class at all? I mean, new constructor which takes parameters like "_name", "_id", "_teacher_id"

Reply:
Removed records parsing outside class.